packages/bpk-docs/src/pages/NeoCardPage/CardPage.js

```diff
@@ -28,7 +28,14 @@ import Native from '../NativeCardsPage';
 const Page = () => (
   <DocsPageWrapper
     title="Neo Card"
```

Comment:
Can you remove `Neo` from the title? (My bad - we discussed it earlier but I forgot to add a comment about it.)